exo/api/chatgpt_api.py

```diff
@@ -171,6 +172,7 @@ def __init__(self, node: Node, inference_engine_classname: str, response_timeout
     cors.add(self.app.router.add_post("/chat/completions", self.handle_post_chat_completions), {"*": cors_options})
     cors.add(self.app.router.add_post("/v1/chat/completions", self.handle_post_chat_completions), {"*": cors_options})
     cors.add(self.app.router.add_get("/v1/download/progress", self.handle_get_download_progress), {"*": cors_options})
+    cors.add(self.app.router.add_get("/modelpool", self.handle_model_support), {"*": cors_options})
```

Comment:
strictly speaking, this doesn't conform to the OpenAI API, which has a `v1/models` endpoint. I see we have a /v1/models above... not sure if that one _does_ conform. 

In any case, it might be good practice to break out the ones that strictly conform and separate them from the ones that don't with a simple comment along the lines of `the endpoints below do not conform to OpenAI standards`. I'd maybe even recommend changing the filename to `rest_api.py` to differentiate, but that's more of an @AlexCheema decision.

Reply:
Totally. I think having a superset of the api spec is reasonable but we could always make another API

Reply:
well, Exo doesn't implement the whole spec, but in any case, it's useful for users to see what conforms to OpenAI and what doesn't, because that answers the biggest question they might have, which is, how do I switch from OpenAI to exo.

Seems a bit out of scope for this PR though.